docs/content/1.getting-started/4.getting-help.md

```diff
@@ -0,0 +1,11 @@
+# Getting Help
+
+At some point, you may find that there's an issue you need some help with.
+
+But don't worry! We're a friendly community of developers and we'd love to help. Concretely this means to:
+- Checkout our docs here,
```

Comment:
Do we want to add a link here?

Reply:
I've clarified a bit, but this is already talking about uitself :D

Reply:
True!